db/seeds.rb

```diff
@@ -202,7 +202,7 @@
                       brand: 'K2',
                       image: File.open('app/assets/images/lien_at.png')
                     },
-                    { name: 'Mission EST Binding',
+                    { name: 'Mission EST',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.